src/OpenTelemetry.Instrumentation.Http/OpenTelemetry.Instrumentation.Http.csproj

```
@@ -12,9 +12,11 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="$(RepoRoot)\src\Shared\EnvironmentVariables\*.cs" Link="Includes\EnvironmentVariables\%(Filename).cs" />
     <Compile Include="$(RepoRoot)\src\Shared\Guard.cs" Link="Includes\Guard.cs" />
-    <Compile Include="$(RepoRoot)\src\Shared\Shims\NullableAttributes.cs" Link="Includes\Shims\NullableAttributes.cs" />
+    <Compile Include="$(RepoRoot)\src\Shared\Options\*.cs" Link="Includes\Options\%(Filename).cs" />
```

nit: this could be simplified
```suggestion
    <Compile Include="$(RepoRoot)\src\Shared\Options\*.cs" LinkBase="Includes\Options" />
```